### DISCUSSION TITLE
anyone try to build on MacOSX?

### DISCUSSION THREAD

**Discussion by ghost** [aristocratos/btop#63] [General]
I got it build and running on my amzn linux ec2 server and it is beautiful!  Nice nice work.

I'm psyched to get it working on mac laptop... getting errors building.

Any thoughts?  I don't think any of the listed binaries would work on Mac because they're linux only...

sean@levanter ➜  btop-1.0.12  ★ seancli ✦ make
find: src/osx: No such file or directory
 
 ██████╗ ████████╗ ██████╗ ██████╗
 ██╔══██╗╚══██╔══╝██╔═══██╗██╔══██╗   ██╗    ██╗
 ██████╔╝   ██║   ██║   ██║██████╔╝ ██████╗██████╗
 ██╔══██╗   ██║   ██║   ██║██╔═══╝  ╚═██╔═╝╚═██╔═╝
 ██████╔╝   ██║   ╚██████╔╝██║        ╚═╝    ╚═╝
 ╚═════╝    ╚═╝    ╚═════╝ ╚═╝      Makefile v1.2
PLATFORM   ?| OSX
ARCH       ?| x86_64
CXX        ?| c++ (4.2.1)
THREADS    :| 4
REQFLAGS   !| -std=c++20
WARNFLAGS  :| -Wall -Wextra -pedantic
OPTFLAGS   :| -O2 -ftree-loop-vectorize -flto=4
LDCXXFLAGS :| -pthread -D_FORTIFY_SOURCE=2 -D_GLIBCXX_ASSERTIONS -fexceptions -fstack-protector -fstack-clash-protection -fcf-protection
CXXFLAGS   +| $(REQFLAGS) $(LDCXXFLAGS) $(OPTFLAGS) $(WARNFLAGS)
LDFLAGS    +| $(LDCXXFLAGS) $(OPTFLAGS) $(WARNFLAGS)

Building btop++ (v1.0.12)
Compiling src/btop.cpp
Compiling src/btop_draw.cpp
Compiling src/btop_input.cpp
clang: error: unknown argument: '-fstack-clash-protection'
clang: error: unknown argument: '-fstack-clash-protection'
clang: error: unknown argument: '-fstack-clash-protection'
clang: error: unknown argument: '-ftree-loop-vectorize'
clang: error: unsupported argument '4' to option 'flto='
clang: error: unknown argument: '-ftree-loop-vectorize'
clang: error: unsupported argument '4' to option 'flto='
clang: error: unknown argument: '-ftree-loop-vectorize'
clang: error: unsupported argument '4' to option 'flto='
make: *** [obj/btop.o] Error 1
make: *** Waiting for unfinished jobs....
make: *** [obj/btop_draw.o] Error 1
make: *** [obj/btop_input.o] Error 1
Compiling src/btop_tools.cpp
clang: error: unknown argument: '-fstack-clash-protection'
clang: error: unknown argument: '-ftree-loop-vectorize'
clang: error: unsupported argument '4' to option 'flto='
make: *** [obj/btop_tools.o] Error 1
sean@levanter ➜  btop-1.0.12  ★ seancli ✦

**Comment by gandol** [comment]
i got same issue but i try to install gcc usig home brew and the problem has gone

**Comment by ghost** [comment]
Thx.  I'll try that.
